include/aikido/constraint/outcome/CollisionFreeOutcome.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef AIKIDO_CONSTRAINT_COLLISIONFREEOUTCOME_HPP_
+#define AIKIDO_CONSTRAINT_COLLISIONFREEOUTCOME_HPP_
+
+#include <vector>
+#include "TestableOutcome.hpp"
+#include "dart/collision/CollisionObject.hpp"
+#include "dart/collision/Contact.hpp"
+#include "dart/dynamics/BodyNode.hpp"
+#include "dart/dynamics/ShapeFrame.hpp"
+#include "dart/dynamics/ShapeNode.hpp"
+
+namespace aikido {
+namespace constraint {
+
+class CollisionFreeOutcome : public TestableOutcome
```

Comment:
Could we add a simple docstring for this class like saying this class is a testable outcome for `CollisionFree` class?